keyboard.js

```diff
@@ -0,0 +1,234 @@
+import { buttons } from './buttons.js'
+
+class Keyboard {
+  constructor () {
+    this.buttons = buttons
+    this.locale = localStorage.locale || 'en'
+    this.caps = false
+    this.CapsLock = false
+    this.Shift = false
+  }
+
+  init () {
+    document.body.insertAdjacentHTML('beforeend',
+      '<section class="virtual_keyboard" id="main">' +
+      '<textarea name="textarea" class="textarea" id="editor" cols="100" rows="10">' +
+      '</textarea>' +
+      '<div class="keyboard" id="keyboard">' +
+      '<div class="line_1" id="line_1">' +
+      '</div>' +
+      '<div class="line_2" id="line_2">' +
+      '</div>' +
+      '<div class="line_3" id="line_3">' +
+      '</div>' +
+      '<div class="line_4" id="line_4">' +
+      '</div>' +
+      '<div class="line_5" id="line_5">' +
+      '</div>' +
+      '</div>' +
+      '</section>'
+    )
+    this.buttons.forEach(button =>
+      button.render(this.locale, false, this.buttons.indexOf(button), this.CapsLock)
+    )
+
+    this.main = document.getElementById('main')
+    this.editor = document.getElementById('editor')
+    this.main.addEventListener('click', (this.clickHandle).bind(this))
+    document.body.addEventListener('keydown', (this.keyHandle).bind(this))
+    document.body.addEventListener('keypress', (this.keyHandle).bind(this))
+    document.body.addEventListener('keyup', (this.keyHandle).bind(this))
+    this.showInfo()
+  }
+
+  showInfo () {
+    if (document.querySelector('#information') === null) {
+      this.editor.insertAdjacentHTML('afterend',
+        '<div class="info__container">' +
+        '<span class="info__text" id="information">' +
+        '</span>' +
+        '</div>'
+      )
+    }
+    const INFO = document.querySelector('#information')
```

Comment:
constants is commonly written in Uppercase if they really doesn't change (const LANGUAGE = 'ru') in your case it is better to use lowercase(camelCase)
`const info = document.querySelector('#information')`